common/spec/dependabot/metadata_finders/base/changelog_finder_spec.rb

```diff
@@ -798,18 +798,9 @@
             "* Fixed `issue <https://github.com/pytest-dev/"\
             "pytest-selenium/issues/216>`_ with TestingBot local tunnel."
           end
-          let(:converted_text) do
-            "1.16.0 (2019-02-12)\n"\
-            "===================\n"\
-            "\n"\
-            "-   `pytest-selenium` now requires pytest 3.6 or later.\n"\
-            "-   Fixed [issue](https://github.com/pytest-dev/"\
-            "pytest-selenium/issues/216) with TestingBot local tunnel.\n"
-          end
 
-          it "converts the rst properly (or falls back)" do
-            expect([converted_text, unconverted_text]).
-              to include(changelog_text)
+          it "does not convert the rst" do
+            expect(changelog_text).to eq(unconverted_text)
```

Comment:
I suspect this is not needed, but thought it would be useful to leave some breadcrumbs in our history. Is it better to remove this test altogether?

Reply:
I kind of like this. If tests are there to assert behaviors or constraints, and we are explicitly deciding on the behavior of _not_ converting RST, it makes sense to assert as much. Also, if someone asks if RST conversion is broken, we have a test to point to :)